content/auth/revocation.textile

```diff
@@ -109,6 +109,25 @@ ablyRest.request('post', '/keys/{{API_KEY_NAME}}/revokeTokens', null, requestBod
 );
 ```
 
+```[rest_python]
+rest = AblyRest(key='{{API_KEY}}')
+
+request_body = {
+    'targets': ['clientId:client1@example.com']
+}
+
+response = await rest.http.post('/keys/{{API_KEY_NAME}}/revokeTokens', body=request_body)
```

Comment:
`rest.http.post` is an internal API, we should be using `rest.request` for this

Reply:
https://github.com/ably/docs/pull/2129/commits/bef897c08a2d403b9f95e836fd3baefa86b8f2cf